### PR TITLE
Update for Omega

### PR DESCRIPTION
**Pull request by garbear**
## Description

This PR updates CI and build files for Omega compatibility.

## How has this been tested?

Passes Win32 and WIn64 Azure CI. Fails UWP-Win64 and UWP-ARM.

Runtime tested:

screenshot00000

Still getting the blue/pink washed out Colors on my Windows 4K TV.

### CODE REVIEW

**Review by VelocityRa — APPROVED** [review]
Build changes LGTM.
UWP seems to fail because it disallows `GetModuleHandle` for some reason, which is used [here](https://github.com/kodi-game/game.shader.presets/blob/920ed85c47bf4eb44317b3056dcdc1844feff502/src/depends/file_path_special.c#L222). MS really went overboard with the sandboxing there...

Regarding the color issue, seems to me like the R and G channels are swapped, and the B channel is set to `0xFF`s (probably swapped with the alpha channel).
So, some code somewhere must assume an `ARGB` surface format instead of `BGRA` (or the inverse).

**Comment by garbear** [comment]
There's been about 6 years of changes to the upstream shader preset code. Is it worth doing the port again? I'm sure UWP has been added upstream by now.

Channels must be getting swapped somewhere. This happens on master too, and I'm not sure where to start hunting down the format mixup.

File: debian/copyright [hunk 1]
```diff
@@ -3,7 +3,7 @@ Upstream-Name: game.shader.presets
 Source: https://github.com/kodi-game/game.shader.presets
 
 Files: *
-Copyright: 2014-2020 Team Kodi
+Copyright: 2014-2021 Team Kodi
```
Comment (gusandrianos) [inline debian/copyright]: Why not 2023 here?

**Review by gusandrianos — APPROVED** [review]
LGTM

File: debian/copyright (hunk 1 above)
Comment (garbear) [reply debian/copyright]: I copied from game.libretro without much brain juice going into it. Thanks for catching, will update.

**Pull request merged**